package.json

```diff
@@ -34,7 +33,8 @@
     "socket.io": "^4.1.2",
     "storybook": "^6.2.9",
     "ts-node": "^10.0.0",
-    "tsnodemon": "^1.2.2"
+    "tsnodemon": "^1.2.2",
+    "reveal-md": "^5.1.1"
```

Comment:
why are these moving around? doesn't look like we needed to touch this file at all